tests/beamlines/unit_tests/test_device_helpers.py

```diff
@@ -0,0 +1,23 @@
+from random import randint
+
+from dodal.beamlines._device_helpers import numbered_slits
+
+
+def test_slit_field_naming():
+    num = randint(1, 20)
```

Comment:
In general I really don't like seeing `random` in tests - it's a recipe for flaky tests later.

If we want to test various cases, we can parameterize the test with e.g. `1, 2, 20`.

Reply:
Ok, parameterized it is